pkg/common/consumer/consumer_manager.go

```diff
@@ -111,6 +134,38 @@ func NewConsumerGroupManager(logger *zap.Logger, serverHandler controlprotocol.S
 	return manager
 }
 
+// AddNotification creates a new ManagerEvent channel and returns it.  Manager events will be
+// broadcast to all channels created this way.
+func (m *kafkaConsumerGroupManagerImpl) AddNotification() <-chan ManagerEvent {
+	eventChan := make(chan ManagerEvent)
+	m.eventLock.Lock()
+	m.notifyChannels = append(m.notifyChannels, eventChan)
+	m.eventLock.Unlock()
+	return eventChan
+}
+
+// ClearNotifications closes and removes all channels from the notification list
+func (m *kafkaConsumerGroupManagerImpl) ClearNotifications() {
+	for _, eventChan := range m.notifyChannels {
```

Comment:
This doesn't appear to be used in the production code - is it just here for API "completeness" ?  OK if so, wasn't sure.

Reply:
Hm, you're right.  I thought I called that in the shutdown code but I seem to have forgotten it.  I'll put it where it belongs.